kstreams-live-update/README.md

```diff
@@ -55,11 +54,17 @@ docker-compose exec kafka /kafka/bin/kafka-console-consumer.sh \
     --bootstrap-server kafka:9092 \
```

Comment:
The paragraph above still referes to temparature

Reply:
Done.